userguide/storage.rst

```diff
@@ -18,6 +18,8 @@ these options:
 * :ref:`Importing a Disk`: import a **single** disk that is
   formatted with the UFS, NTFS, MSDOS, or EXT2 filesystem.
 
+* :ref:`Swap Memory`: Change the swap size.
```

Comment:
This is probably best known as "Swap Space".  "Memory" generally is understood to be RAM.
```suggestion
* :ref:`Swap Space`: Change the swap space size.
```